tools/compiler/main.rs

```diff
@@ -1,46 +1,41 @@
 // Copyright © SixtyFPS GmbH <info@sixtyfps.io>
 // SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
 
+use clap::Parser;
 use sixtyfps_compilerlib::diagnostics::BuildDiagnostics;
 use sixtyfps_compilerlib::*;
 use std::io::Write;
-use structopt::StructOpt;
 
-#[derive(StructOpt)]
+#[derive(Parser)]
```

Comment:
```suggestion
#[derive(clap::Parser)]
```

Because I think the parser term is ambiguous with our own parser

Reply:
Good point, I'll change that!